src/web/routes/application/common/confirmation-code.js

```diff
@@ -0,0 +1,16 @@
+// reset confirmation code and reset state to in progress so that the user is not taken straight back to check details.
+const { CONFIRMATION_CODE_ENTERED_SESSION_PROPERTY } = require('./constants')
+const { stateMachine, states } = require('../common/state-machine')
+
+const handleConfirmationCodeReset = req => {
+  req.session[CONFIRMATION_CODE_ENTERED_SESSION_PROPERTY] = false
```

Comment:
Not for this PR, but I think some of the higher level constants files could be split out into distinct modules (e.g. paths, session keys etc..).